packages/graph/hash_graph/bench/benches/representative_read/seed.rs

```diff
@@ -135,16 +136,14 @@ async fn seed_db(account_id: AccountId, store_wrapper: &mut StoreWrapper) {
             .id()
             .clone();
 
-        let mut entity_ids = Vec::with_capacity(quantity);
-        for _ in 0..quantity {
-            entity_ids.push(
-                store
-                    .create_entity(entity.clone(), entity_type_id.clone(), account_id, None)
-                    .await
-                    .expect("failed to create entity")
-                    .entity_id(),
-            );
-        }
+        store
+            .create_entities(
+                repeat(None).zip(repeat(&entity).take(quantity).cloned()),
```

Comment:
It seems that `repeat` requires clone. Can you not just do this?

```suggestion
                repeat((None, entity)).take(quantity),
```